plone/supermodel/model.py

```diff
@@ -107,8 +107,22 @@ def finalizeSchemas(parent=Schema):
         )
 
     def walk(schema):
-        yield schema
-        for child in schema.dependents.keys():
+        # When we have behaviors on the Plone site root we got some shcmeas that 
+        # are not SchemaClasses
+        if isinstance(schema, SchemaClass):
+            yield schema
+
+        # This try..except is to handle AttributeError:
+        # 'VerifyingAdapterLookup' object has no attribute 'dependents'.
+        # afaik this happens in tests only.
+        # We have issue https://github.com/plone/plone.supermodel/issues/14
+        # to find out why this is happening in the first place.
+        try:
+            children = schema.dependents.keys()
+        except AttributeError:
```

Comment:
It seems this branch is needed to fix the plip branch build.
After I added this commit:
- https://github.com/plone/buildout.coredev/commit/f5c817d5e638a5f6b1651177f47db88fa3b10543
I had a succesful build:
- https://jenkins.plone.org/job/plip-dx-siteroot-3.9/5/

Before I started:
- https://jenkins.plone.org/view/PLIPs/job/plip-dx-siteroot-3.7/31/
which is not yet over but I expect it to fail

Reply:
It was my mistake to remove this checkout from the PLIP.
I probably got confused between the various packages. Sorry!

Since the job of this individual PR is green on 6.0, and I already let 5.2 use a maintenance branch, it would be fine to merge this PR now, without the other PLIP PRs.
@ale-rt Do you want to push the merge button?